PersonaStats.Rmd

```diff
@@ -0,0 +1,294 @@
+---
+title: "Personas"
+author: "Kate Isaac"
+date: "`r Sys.Date()`"
+output: html_document
+---
+
+```{r}
+library(tidyverse)
+library(dplyr)
+library(data.table)
+library(magrittr)
+library(here)
```

Comment:
```suggestion
library(dplyr)
library(magrittr)
```
I think these are not needed.. worth checking with the `knitr::purl` added below.